hooks/providers/use-provider-me.ts

```
@@ -0,0 +1,36 @@
+  const { isSignedIn } = useAuth();
+
+  return useQuery<ApiProvider | null>({
+    queryKey: queryKeys.provider.me(),
+    queryFn: async () => {
+      if (!isSignedIn) return null;
+      
+      try {
+        const apiClient = await getApiClient();
+        const response = await apiClient.get('/providers/me');
+        return response.data;
+      } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)